### ISSUE TITLE
Typo in length example

### ISSUE THREAD

**Issue by pyrocto** [vekatze/neut#16]
```
-- length-internal : (tau, noema (list a)) -> i64
length-internal (a : tau) (xs : noema (list a)) :=
  case-noetic xs of
    nil ->
      0
    cons y ys ->
      length-internal a ys
```
You need to add one on the last line.

**Comment by pyrocto** [comment]
Fix in #17

**Comment by vekatze** [comment]
Good catch. Thanks for your correction. I approved it.

By the way, I'm currently in the process of replacing the readme with the [website](https://vekatze.github.io/neut/) (currently under heavy construction). Because of that, I think I'll have to clear this correction in near future. I would appreciate your understanding...
